universal-login-react/package.json

```diff
@@ -16,11 +16,13 @@
   },
   "dependencies": {
     "@ramp-network/ramp-instant-sdk": "^1.0.1",
+    "@restless/sanitizers": "^0.2.2",
     "@universal-login/commons": "0.3.40",
     "@universal-login/sdk": "0.3.40",
     "dotenv": "^8.1.0",
     "ethers": "^4.0.36",
     "lodash.range": "^3.2.0",
+    "node-fetch": "^2.6.0",
```

Comment:
Do we really want node-fetch as a dependency in react project?

Reply:
It's necessary only for test purposes. Moved to dev dependencies.